docs/how-to/tensorboard.txt

```diff
@@ -51,6 +51,20 @@ TensorBoard for multiple experiments use
   metrics from persistent storage. It may take up to 5 minutes for TensorBoard
   to receive data and render visualizations.
 
+Specifying a Command Config
+---------------------------
+
+Determined supports initializing TensorBoard with a config file. This
+feature can be useful for running TensorBoard with a specific container
+image or with a different resource allocation. Configuration settings
+can be found in the  :ref:`command-notebook-configuration`.
```

Comment:
Can we update the command-notebook-configuration doc to specify how that configuration can be applied to TensorBoards as well?

Reply:
I added a clarification in the command-notebook-configuration, but everything there should be compatible with tensorboard.